src/algebra/group/hom.lean

```diff
@@ -706,6 +706,13 @@ def comp_hom [mul_one_class M] [comm_monoid N] [comm_monoid P] :
   map_one' := by { ext1 f, exact one_comp f },
   map_mul' := λ g₁ g₂, by { ext1 f, exact mul_comp g₁ g₂ f } }
 
+/-- Flipping arguments of monoid morphisms (`monoid_hom.flip`) as a monoid morphism. -/
+@[to_additive "Flipping arguments of additive monoid morphisms (`add_monoid_hom.flip`)
+as an additive monoid morphism.", simps]
+def flip_hom {mM : monoid M} {mN : monoid N} {mP : comm_monoid P}
```

Comment:
I think this probably works with just `mul_one_class` instead of `monoid`?

Reply:
Indeed, thanks!

I'd based the arguments on those of `monoid_hom.flip`, but I was originally working from a version of master that was a few days old and didn't have 5f1b4500f21b62968aad9bd939625cdf2b2372a6. So basically this is a rebase error (of the at-a-distance kind that Git can't automatically spot.) Fixing.